packages/flutter/lib/src/material/menu_anchor.dart

```diff
@@ -582,9 +574,6 @@ _MenuAnchorState? get _previousFocusableSibling {
     if (!_isOpen) {
       return;
     }
-    if (_isRoot) {
-      FocusManager.instance.removeEarlyKeyEventHandler(_checkForEscape);
-    }
```

Comment:
At some point in the past, there was a key event handler that used this function:

```dart
if (_isRoot) {
  FocusManager.instance.addEarlyKeyEventHandler(_checkForEscape);
}
```

But then it was removed in #130534.

Usually, these private methods are really good about warning you when they're never used, but this was one of those times where it slipped under the radar (since technically it has a reference).

Reply:
Thanks for the context!